### PR TITLE
Add eslint/prettier with airbnb base with typescript integration

### LINKED ISSUE #1: Setup and config eslint and prettier
use eslint airbnb please

### PR DESCRIPTION
**Pull request by nabeel-**
Closes #1 

I opted to disable any errors that weren't obvious fixes and leave the warnings for now since they won't actually fail the eslint run. Happy to reconfigure/add/remove any rules!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "build": "next build",
     "start": "next start",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "test": "eslint . --ext=tsx,ts"
```
Comment (mmahalwy) [inline package.json:12]: can you create a separate script for this to be just `lint` and `test` would be `lint` and soon to be jest tests too :)

**Comment by mmahalwy** [comment]
wow @nabeel- thanks so much for the quick turn around :)

File: package.json (hunk 1 above)
Comment (nabeel-) [reply package.json]: Done!

**Comment by mmahalwy** [comment]
@nabeel- got a few more issues if you're up for it :)

**Pull request merged**